CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+### citus v9.0.0 (October 7, 2019) ###
+
+* Adds support for PostgreSQL 12
+
+* Adds UDFs to help with PostgreSQL upgrades
+
+* Delegates `SELECT function()` to a worker node
+
+* Distributes types to worker nodes
+
+* Introduces `create_distributed_function` UDF
+
+* Introduces local query execution for Citus MX
+
+* Implements infrastructure for routing `CALL` to MX workers
+
+* Adds a feature flag to turn off `CREATE TYPE` propagation
+
+* Adds option `citus.single_shard_commit_protocol`
+
+* Adds schemas when distributing sequence definitons correctly
```

Comment:
Shouldn't we say that this is `Fixes a bug ..`

Reply:
I replaced it with `Fixes a bug in schemas of distributed sequence definitions`